### DISCUSSION TITLE
Split depths between non-cored (soil) and cored (rock) logs

### DISCUSSION THREAD

**Discussion by SolidRockProgrammer** [australiangeomechanics/agsformat#9] [Ideas]
AGS 4.1.0 AU 1.x doesn't have heading to define the split between a non-cored and cored borehole log.  I can't recall if we ever discussed it during a committee meeting.

It is easy enough to assume the first depth on the CORE group is the spit depth, so i don't see this is a big problem.

But what if you are working in Melbourne, and you want to split back and forth between the logs many times for one LOCA (borehole).  Do you change drilling methods at the boundary (Core to Wash bore), so you can use the CORE table Depth and Bottom, or do we need a group to define this information?

Another thought is to base it on data in HDPH_TOP, HDPH_TOP, HDPH_METH. Again, it assumes that soil is always wash bored and rock is always cored.

Thoughts?

**Comment by digitalgeotechnics** [comment]
@SolidRockProgrammer I'm not sure we discussed this or not either, but is this rendering information?

I've found it more favourable to orientate log templates around the type of material encountered rather than the method - as soil can be cored and rock can be wash bored.  If you use that approach you can use the GEOL_TYPE (? can't remember the heading off the top of my head) field to dictate switches between templates.  Although some thought needs to go into handling XW bands of soil.  It gets pretty impractical to switch between templates just to cater to a 100mm band of soil.  especially when there are several of them.

**Reply by SolidRockProgrammer** [reply]
We should use GEOL_TYPE to define if data is displayed on a SOIL log or ROCK logs
